pkg/comp-functions/functions/vshnpostgres/restart.go

```diff
@@ -89,6 +94,16 @@ func getPendingRestart(ctx context.Context, svc *runtime.ServiceRuntime) (time.T
 	return time.Time{}, nil
 }
 
+// In case the operator was updated and the PostgreSQL clusters requires immediate restart we will postpone it during the maintenance
```

Comment:
```suggestion
// In case the operator was updated and the PostgreSQL clusters require a security maintenance, we ensure that it only happens during the maintenance window
```